docs/CONTRIBUTING.md

```diff
@@ -195,13 +195,13 @@ Once built each package is released on [Github](https://github.com/insitu-projec
 
 > The publication on the Chrome store may take a while -- days -- to be validated.
 
-> See the detailed `staging` deploy steps `./release.config.staging.js` in project root directory.
+> See the detailed `staging` deploy steps `./release.config.js` in project root directory.
```

Comment:
Might add a few info on how to release on staging and how it works with Semaphore.